### DISCUSSION TITLE
Browser extension for GitHub

### DISCUSSION THREAD

**Discussion by brynary** [qltysh/qlty#1171] [Feature Requests]
### Problem statement

I would like to view code quality data like coverage directly on GitHub 

### Solution brainstorm (optional)

Browser extension for Chrome, Firefox, Safari which renders the Qlty data onto GitHub.com

**Comment by brynary** [comment]
This is in development!

**Comment by brynary** [comment]
OK, we just pushed the initial version of the Chrome extension public: https://github.com/qltysh/qlty-browser

We're calling it an alpha today. You can install it using the instructions in the repository, because it's not in the Chrome Web Store yet.

Please let us know what you think!

**Reply by isaachinman** [reply]
Thanks @brynary! We previously relied heavily on the CodeClimate extension when reviewing GitHub PRs. Any idea when this will be available in the Chrome Web Store?
